tools/generators/migrate-converged-pkg/index.ts

```diff
@@ -711,11 +712,10 @@ function setupStorybook(tree: Tree, options: NormalizedSchema) {
   return tree;
 }
 
-function moveStories(tree: Tree, options: NormalizedSchema) {
-  const componentName = names(options.normalizedPkgName).className.replace('React', '');
+function moveStoriesToPackageRoot(tree: Tree, options: NormalizedSchema) {
```

Comment:
**chore:** we need to update also `.storybook/tsconfig.json` otherwise we will loose TS/DX completely

Reply:
Updated `.storybook/tsconfig.json` in [240795d](https://github.com/microsoft/fluentui/pull/24458/commits/240795ddee005326a39cf1f7285924417bb4dab1). Curious to know what needs to be updated for `tsconfig.lib.json`? 🤔

Also good call on the eslint configuration! The `stories/` folder loses linting capabilities because our eslint configuration only lints the `src/` folder. Ive made the change in d31acbb to also lint the `stories/` folder if it exists.